irispy/iris_tools.py

```diff
@@ -246,7 +246,7 @@ def get_iris_response(pre_launch=False, response_file=None, response_version=Non
 
 
 @custom_model
-def _gaussian1d_on_linear_bg(x, amplitude=None, mean=None, standard_deviation=None,
+def _gaussian1d_on_linear_bg(x=None, amplitude=None, mean=None, standard_deviation=None,
```

Comment:
This function should not be called directly.  It's only used as part of the fitting routine in ```_calculate_orbital_wavelength_variation()```.  So you don't need to test this line, if that's what you're trying to do.